dev/packages/alpha/cisco-0.0.2/dataset/asa/manifest.yml

```diff
@@ -0,0 +1,59 @@
+title: Cisco ASA logs
+release: ga
+type: logs
+streams:
+  - input: syslog
+    title: Cisco ASA logs
+    description: Collect Cisco ASA logs
+    template_path: stream.yml
+    vars:
+      - name: tags
+        type: text
+        title: Tags
+        multi: true
+        required: true
+        show_user: true
+        default:
+          - cisco-asa
+      - name: syslog_port
+        type: integer
+        title: Syslog Port
```

Comment:
Is there a configuration option for `Syslog Host` or it's only port as it's localhost? Maybe a description would clarify this.